src/app.js

```diff
@@ -1,42 +1,36 @@
-'use strict';
-
-define(function (require) {
-  require('audioworklet-polyfill');
-  require('shims');
-  require('audiocontext');
-  var p5SOUND = require('master');
-  require('helpers');
-  require('errorHandler');
-  require('audioWorklet');
-  require('panner');
-  require('soundfile');
-  require('amplitude');
-  require('fft');
-  require('signal');
-  require('oscillator');
-  require('envelope');
-  require('pulse');
-  require('noise');
-  require('audioin');
-  require('filter');
-  require('eq');
-  require('panner3d');
-  require('listener3d');
-  require('delay');
-  require('reverb');
-  require('metro');
-  require('looper');
-  require('soundLoop');
-  require('compressor');
-  require('soundRecorder');
-  require('peakDetect');
-  require('gain');
-  require('monosynth');
-  require('polysynth');
-  require('distortion');
-  require('audioVoice');
-  require('monosynth');
-  require('polysynth');
-
-  return p5SOUND;
-});
+import 'audioworklet-polyfill';
```

Comment:
for a future PR, this may be an opportunity to decouple p5.sound from p5. In the app module, we could check to see if p5 exists on the page. If it doesn't, we could behave differently, allowing p5.sound to be a stand-alone module. Just a thought...